File: src/main/java/me/codedlife/newjessica/NewJessica.java
package me.codedlife.newjessica;

import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;


public class NewJessica implements ModInitializer {

    private final static String ID = "NewJessica";
    private final static Logger LOGGER = LoggerFactory.getLogger( ID );

    @Override
    public void onInitialize() {
        LOGGER.info( "NewJessica mod initialized" );
    }
}

File: src/main/kotlin/org/jessixperience/jessica/NewJessica.kt
package org.jessixperience.jessica
import net.fabricmc.api.ModInitializer
import org.jessixperience.jessica.utils.AutoOffHand
import org.jessixperience.jessica.utils.interfaces.Util
import org.slf4j.Logger
import org.slf4j.LoggerFactory


object NewJessica: ModInitializer
{
    const val MOD_ID = "NewJessica"
    const val RELEASE = "0.0.2"

    val LOGGER: Logger = LoggerFactory.getLogger( MOD_ID )
    val utils: List<Util> = listOf(
        AutoOffHand()
    )

    fun getLogger(): Logger {
        return LOGGER;
    }

    override fun onInitialize() {

        println("$MOD_ID $RELEASE initialized. Let's fun")
    }
}